packages/docs-markdown/src/controllers/yaml/showTOCQuickPick.ts

```diff
@@ -1,20 +1,30 @@
 import { createEntry } from './createEntry';
 import { readFileSync } from 'fs';
-import { basename, dirname, join, relative } from 'path';
+import * as recursive from 'recursive-readdir';
+import { basename, dirname, extname, join, relative } from 'path';
 import { QuickPickItem, window, workspace } from 'vscode';
-import { noActiveEditorMessage, postError, showStatusMessage } from '../../helper/common';
+import { ignoreFiles, noActiveEditorMessage } from '../../helper/common';
 import { noHeadingSelected } from '../../constants/log-messages';
 import { getHeadings } from '../../helper/getHeader';
-import fg = require('fast-glob');
 
 export async function showTOCQuickPick(options: boolean) {
+	const markdownExtensionFilter = ['.md'];
 	let folderPath: string = '';
+	let fullPath: string = '';
 
 	if (workspace.workspaceFolders) {
 		folderPath = workspace.workspaceFolders[0].uri.fsPath;
 	}
 
-	const items = getQuickPickItems(folderPath);
+	const files = await recursive(folderPath, ignoreFiles);
+
+	const items: QuickPickItem[] = [];
+	files.sort();
+	files
+		.filter((file: any) => markdownExtensionFilter.indexOf(extname(file.toLowerCase())) !== -1)
+		.forEach((file: any) => {
```

Comment:
```suggestion
		.filter((file: string) => markdownExtensionFilter.indexOf(extname(file.toLowerCase())) !== -1)
		.forEach((file: string) => {
```

If these are of `string` type, it should be explicit - saying they're `any` is rather misleading.